src/services/ghost/classic-auto-complete/GhostInlineCompletionProvider.ts

```diff
@@ -129,8 +128,6 @@ export class GhostInlineCompletionProvider implements vscode.InlineCompletionIte
 	 * @returns LLM retrieval result with suggestions and usage info
 	 */
 	public async getFromLLM(context: GhostSuggestionContext, model: GhostModel): Promise<LLMRetrievalResult> {
-		this.isRequestCancelled = false
```

Comment:
the existence of this code makes the (wrong) assumption that we can only make one request at the time